### ISSUE TITLE
Tt cannot find src/main/java/caffe/Caffe.java ，where is Caffe.java？

### ISSUE THREAD

**Issue by benbenyao** [yahoo/CaffeOnSpark#275]
protoc:
     [exec] make[1]:  `/root/CaffeOnSpark/caffe-distri'
     [exec] make[1]: `/root/CaffeOnSpark/caffe-distri'
     [exec] make[1]: *** “src/main/java/caffe/Caffe.java”“../caffe-public/distribute/proto/caffe.proto”。

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.7:run (proto) on project caffe-distri: An Ant BuildException has occured: exec returned: 2
[ERROR] around Ant part ...<exec failonerror="true" dir="/root/CaffeOnSpark/caffe-distri" executable="make">... @ 5:84 in /root/CaffeOnSpark/caffe-distri/target/antrun/build-protoc.xml
[ERROR] -> [Help 1]

**Comment by junshi15** [comment]
That's a generated file by protoc. It sounds like you had problems with protobuf.

**Comment by benbenyao** [comment]
That's my protobuf --version：
root@ubuntu001:/etc/ld.so.conf.d# protoc --version
libprotoc 2.5.0
And i think, could me use BLVC's caffe？I do this, ln -s /home/alpha/caffe caffe-public，but problm is on。

**Comment by junshi15** [comment]
no, using BVLC caffe won't work. caffe-public is based on BVLC caffe, but with modifications.
